contracts/contracts/vault/VaultCore.sol

```diff
@@ -96,6 +105,31 @@ contract VaultCore is VaultStorage {
         }
     }
 
+    /**
+     * @dev Mint OUSD for OUSD Meta Strategy
+     * @param _amount Amount of the asset being deposited
+     *
+     * Notice: can't use nonReentrant modifier since BaseCurveStrategy's deposit
+     * already has that modifier present
+     */
+    function mintForStrategy(uint256 _amount)
+        external
+        whenNotCapitalPaused
+        onlyOusdMetaStrategy
+    {
+        require(_amount > 0, "Amount must be greater than 0");
```

Comment:
One safeguard that comes to mind is the ratio of `net_OUSD_in_circulation/dollar_value_of_gauge_OUSD3CRVLP` In most cases I would expect that value to be close to 0.5. Adding a configurable safeguard so that the ratio doesn't go beyond - lets say 0.7 would make sense to me.